src/components/data-entry/TabSlider.js

```diff
@@ -25,9 +25,13 @@ const TabSlider = props => {
   }))
 
   useEffect(() => {
-    const selectedRefIdx = props.options.findIndex(
+    let selectedRefIdx = props.options.findIndex(
       option => option.key === selectedOption
     )
+    if (selectedRefIdx === -1) {
```

Comment:
does this happen?

Reply:
yep, not exactly sure what was triggering it. I think if your current theme is matrix and you lose gold tier, this will crash. I should probably handle that bug the proper way but it feels like such an edge case it's hardly worth it 🤷‍♂️